### PR TITLE
Use master builder.

### PR DESCRIPTION
**Pull request by tmcgilchrist**
Might need to upload the bench results too, not sure?

### CODE REVIEW

**Comment by charleso** [comment]
:+1: With  maybe `HADDOCK`

**Comment by nhibberd** [comment]
These aren't the same, missing publish

**Comment by nhibberd** [comment]
Sorry didn't see the original description, keeping the history is important now (especially given how ci is going to operate in the future).

master has a runner that supports publish bench now as well

**Comment by nhibberd** [comment]
For publishing bench (https://github.com/ambiata/twine/blob/master/test/bench.hs#L45) will need to change as well to `dist/build/twine/twine-bench.html` as per https://github.com/ambiata/master-ambiata#haskell-library

File: master.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[master]
+  runner = "s3://ambiata-dist/master-ambiata/master-haskell-library-0.0.1-20151013064111-d6a3241"
+  version = 1
+  sha1 = "53221a5acfd276d578b1ebb232516d5f42ba4cca"
+
+[build.dist]
+  HADDOCK="true"
+
+[build.dist.bench]
```
Comment (nhibberd) [inline master.toml]: build.dist-bench

File: master.toml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  BENCH="true"
+
+[build.branches]
+
+[build.branches.bench]
```
Comment (nhibberd) [inline master.toml]: build.branches-bench

**Comment by nhibberd** [comment]
Update to latest runner then :100:

**Pull request merged**